### DISCUSSION TITLE
Ajouter un indicateur de durabilité d'une arme/armure

### DISCUSSION THREAD

**Discussion by Nargacaura** [Crownicles/Crownicles#1800] [Suggestions fermées]
Cette idée est un peu farfelue et a des chances d'être refusée, certes, mais ça m'est venu en tête, d'où cette discussion. (À ce que je sache, je n'en ai pas vu qui traîtait de ça, corrigez-moi s'il y en a déjà une dessus.)

Le concept est d'ajouter une certaine durabilité aux armes et armures (les communes étant les plus fragiles, et les mythiques, les plus durables) afin d'ajouter un peu plus de challenge sur la gestion de son inventaire, en sachant qu'il y a de plus en plus de joueurs avec un stuff bien bourrin/tanky, et que le challenge laisserait peut-être place à une certaine lassitude à partir d'un certain moment (j'dis ça, mais c'est juste mon opinion, et je respecte celle de ceux qui pensent le contraire). La durabilité de l'arme/armure diminuera au fil des combats (soit selon le nombre de combats, soit dépendant du nombre de tours), et éventuellement au fil des rapports (mini-events/events) nécessitant l'utilisation d'une arme/armure. Quand une arme ou une armure a son indicateur à 0, elle est considérée comme "brisée" et s'enlève de l'inventaire. Dans ce cas-là, le joueur devra soit trouver une autre, soit équiper celle dans sa réserve.

Éventuellement, cette durabilité pourra probablement être réparée via un forgeron pour la somme de 5k 💰 minimum, directement via le shop. Toutefois, le forgeron ne réparera l'arme/armure que si l'indicateur est à moins de 50% du total de la durabilité de l'équipement.

Si vous avez d'autres options (ou voulez simplement exprimer votre opinion dessus), vos commentaires sont les bienvenus :)

**Comment by BastLast** [comment]
Techniquement hardcore

**Comment by Akutana** [comment]
Nan mais ça va bien les concepts pour tout compliquer ? X)
